### PR TITLE
feat(subjdomain_actions_and_tasks): add atomic and non-atomic action classes

### PR DESCRIPTION
**Pull request by NikiforovSergei**

### CODE REVIEW

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_actions_and_tasks/section_subjdomain_actions_and_tasks.scs [hunk 1]
```diff
@@ -707,6 +753,12 @@ section_subjdomain_actions_and_tasks = [*
                                          {
                                          	independent_actions_class;
                                          	non_independent_actions_class
+                                         };
+
+                                         <= nrel_subdividing:
+                                         {
+                                         	non_atomic_action_class;
```
Comment (KovalM) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_actions_and_tasks/section_subjdomain_actions_and_tasks.scs:760]: format spaces here

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_actions_and_tasks/section_subjdomain_actions_and_tasks.scs [hunk 2]
```diff
@@ -50,6 +50,8 @@ section_subjdomain_actions_and_tasks = [*
                                          	external_subject_ostis_system_interaction;
                                          	external_subject_ostis_system_without_interaction;
                                          	actions_class;
+                                            non_atomic_action_class;
```
Comment (KovalM) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_actions_and_tasks/section_subjdomain_actions_and_tasks.scs:53]: format spaces here

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_actions_and_tasks/content_html/explanation_for_atomic_action_class.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<b><sc_element sys_idtf = "atomic_action_class">Атомарный класс действий</sc_element></b> - <sc_element sys_idtf = "actions_class">класс действий</sc_element>, для которого в данной <sc_element sys_idtf = "ostis_system">ostis-системе</sc_element> существует <sc_element sys_idtf = "sc_agent">sc-агент</sc_element>, способный его выполнить.
```
Comment (KovalM) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_actions_and_tasks/content_html/explanation_for_atomic_action_class.html:1]: Add html formatting

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_actions_and_tasks/content_html/explanation_for_non_atomic_action_class.html [hunk 4]
```diff
@@ -0,0 +1 @@
+<b><sc_element sys_idtf = "non_atomic_action_class">Неатомарный класс действий</sc_element></b> - <sc_element sys_idtf = "actions_class">класс действий</sc_element>, для которого в данной <sc_element sys_idtf = "ostis_system">ostis-системе</sc_element> не существует конкретного <sc_element sys_idtf = "sc_agent">sc-агента</sc_element>, способного его выполнить, и который <sc_element sys_idtf = "nrel_decomposition_of_action">декомпозируется</sc_element> на более простые классы действий (не обязательно атомарные).
```
Comment (KovalM) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_actions_and_tasks/content_html/explanation_for_non_atomic_action_class.html:1]: Add html formatting

**Review by KovalM — CHANGES_REQUESTED** [review]

**Review by KovalM — APPROVED** [review]

**Review by ShunkevichDV — APPROVED** [review]

**Pull request merged**